sections/todo/time_tracker.js

```diff
@@ -313,35 +316,65 @@ var TimeTracker  = class TimeTracker {
         return false;
     }
 
-    _enable_file_monitors () {
-        [this.daily_csv_file_monitor,] = MISC_UTILS.file_monitor(this.daily_csv_file, () => {
-            this._on_tracker_files_modified();
-        });
-
-        [this.yearly_csv_file_monitor,] = MISC_UTILS.file_monitor(this.yearly_csv_file, () => {
-            this._on_tracker_files_modified();
-        });
-
-        [this.yearly_csv_dir_monitor,] = MISC_UTILS.file_monitor(this.yearly_csv_dir, () => {
-            this._on_tracker_files_modified();
-        });
+    _enable_file_monitors (timeout = 100) {
+        try{
+            if (!this.daily_csv_file_monitor){
+                this.daily_csv_file_monitor = this.daily_csv_file.monitor(Gio.FileMonitorFlags.NONE, null);
+            }
+    
+            if (!this.yearly_csv_file_monitor){
+                this.yearly_csv_file_monitor = this.yearly_csv_file.monitor(Gio.FileMonitorFlags.NONE, null);
+            }
+    
+            if (!this.yearly_csv_dir_monitor){
+                this.yearly_csv_dir_monitor = this.yearly_csv_dir.monitor(Gio.FileMonitorFlags.NONE, null);
+            }
+    
+            Mainloop.timeout_add(timeout,()=>{
+                this.daily_csv_file_monitor_id = this.daily_csv_file_monitor.connect('changed', (...args) =>{
+                    this._on_tracker_files_modified();
+                    Main.notify('daily_csv file is modified.')
+                });
+                this.yearly_csv_file_monitor_id = this.yearly_csv_file_monitor.connect('changed', (...args) =>{
+                    this._on_tracker_files_modified();
+                    Main.notify('yearly csv file is modified.')
+                });
+                this.yearly_csv_dir_monitor_id = this.yearly_csv_dir_monitor.connect('changed', (...args) =>{
+                    this._on_tracker_files_modified();
+                    Main.notify('yearly csv dir is modified.')
+                });
+            });
+        }
+        catch (e){
+            logError(e);
+            _disable_file_monitors ();
+            if (this.daily_csv_file_monitor){
+                this.daily_csv_file_monitor.cancell();
```

Comment:
Typo in `cancell`.

Also, please add spaces between various symbols like braces to be consistent with the rest of the file.